project4/project/ebs/views.py

```diff
@@ -0,0 +1,64 @@
+import json
+from django.shortcuts import render
+from django.http import HttpResponse
+from .models import Organisation
+from .forms import UserForm,OrgForm
+from django.contrib.auth.models import User
+from django.forms import ModelForm
+from django.contrib import messages
+from django.contrib.auth.models import Group
+from django import forms
+import sys
+# Create your views here.
+
+def home(request):
+	f1=UserForm();
+	f2=OrgForm();
+	return render(request,'ebs/home.html',{'f1':f1,'f2':f2})
+
+def adminlogin(request):
+	return render(request,'ebs/adminlogin.html')
+
+def forgotpass(request):
```

Comment:
Use the function naming rules: lowercase with words separated by underscores as necessary to improve readability.